src/renderer/components/Message.js

```diff
@@ -54,7 +54,7 @@ class RenderMessage extends React.Component {
 
     const contact = {
       onSendMessage: () => console.log('send a message to', fromId),
```

Comment:
This is never called either since `renderSendMessageButton()` renders nothing due to `contact.hasSignalAccount` is not set.

https://github.com/deltachat/deltachat-desktop/blob/4aceefe4306d611f12d8d92178ee4b334edc616d/conversations/ts/components/conversation/Message.tsx#L625-L627